ui/listner-api/src/main/scala/pl/touk/nussknacker/listner/ChangeEvent.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.listner
```

Comment:
What about moving it to `pl.touk.nussknacker.ui.listener`? Because I assume that that are ui events... Also typo: listner => listEner

Reply:
You're totally right